app/controllers/item.js

```diff
@@ -31,134 +32,141 @@ var UserSourceAuth = require('app/models/userSourceAuth');
 var UserStorageAuth = require('app/models/userStorageAuth');
 var validateParams = require('app/lib/validateParams');
 
+var util= require('util');
+
 var queue = kue.createQueue();
 
-queue.process('storeItemData', function(queueJob, done) {
-  debug('process queueJob %s', queueJob.id);
+var tools = require('app/lib/utils/debuggingTools');
 
-  var getItem = (done) => {
-    Item.findById(queueJob.data.itemId, (error, item) => {
-      if (error) {
-        done(error);
-      } else if (!item) {
-        done(new Error('Item with queueJob.data.itemId not found'));
-      } else {
-        done(undefined, item);
-      }
-    });
-  };
+const STORE_ITEM_DATA = 'storeItemData';
```

Comment:
Because it should never change as it's a string constant -- in general it's better to have these than hard-code strings, even if in a case like this it's a bit overkill as the string was used only once or twice.  Just bothered me…